.travis.yml

```diff
@@ -7,15 +7,15 @@ script:
 
 # Docker Run to Execute Unit Test Package
 - echo "Running Unit Tests"
-- docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS todo-app:test Test/view_model_test.py
+- docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS -e MONGODB_CONNECTION_STRING todo-app:test Test/view_model_test.py
 
 # Docker Run to Execute Unit Integration Package 
 - echo "Running Integration Tests"
-- docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS todo-app:test Test/client_test.py
+- docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS -e MONGODB_CONNECTION_STRING todo-app:test Test/client_test.py
 
 # Docker Run to Execute Unit E2E Selenium Package
 - echo "Running E2E Tests"
-- docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS -e ANON_USER -e ANON -e SECRET_KEY todo-app:test Test/selenium_test.py
+# - docker run -e board_id -e list_id_Done -e list_id_Pending -e list_id_ToDo -e MONGO_LOGIN -e MONGO_PASS -e MONGODB_CONNECTION_STRING -e ANON_USER -e ANON -e SECRET_KEY todo-app:test Test/selenium_test.py
```

Comment:
Why has this been commented out? It should use a non-production version of your database (like your original MongoDB) but we do still want to be running the e2e tests